src/commands/deployAzurePipeline/configureCicdPipeline/configurePipeline.ts

```diff
@@ -0,0 +1,47 @@
+import * as vscode from 'vscode';
+import * as k8s from 'vscode-kubernetes-tools-api';
+import { IActionContext } from 'vscode-azureextensionui';
+import { resourceNode } from './models/resourceNode';
+const CONFIGURE_PIPELINE_COMMAND = 'configure-cicd-pipeline';
+const DEPLOY_TO_AZURE_EXTENSION_ID = 'ms-vscode-deploy-azure.azure-deploy';
+
+export async function configurePipeline(context: IActionContext, target: any): Promise<void> {
+    const deployToAzureExtensionInstalled = await isDeployToAzureExtensionInstalled();
```

Comment:
💡 :) since its no longer an `async`, this specific `await` is redundant. (you forgot this place) https://github.com/Azure/vscode-aks-tools/pull/42/commits/e162e3de51ec5d686421fdefc56c24655adf308b

Reply:
ohho!
yeah, updated. thanks :)